e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-cleanup.yml

```diff
@@ -23,12 +23,12 @@
     app: percona
 
 - name: Confirm pvc pod has been deleted
-  shell: source ~/.profile; kubectl get pods | grep {{ pvc.stdout }}
+  shell: source ~/.profile; kubectl get pods
   args:
     executable: /bin/bash
   delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
   register: result
-  until: "'pvc' and 'Running' not in result.stdout"
+  until: "pvc.stdout not in result.stdout"
```

Comment:
See that the rest of the playbooks have a {{pv.stdout}} convention - is this above ok?

Reply:
Modified